### DISCUSSION TITLE
Linux aarch64 build?

### DISCUSSION THREAD

**Discussion by robby-phd** [cvc5/cvc5#10211] [General]
Is there any plan on providing a Linux arm64 binary in upcoming releases?

**Comment by daniel-larraz** [comment]
Sorry for the late reply. We've just started to publish arm64 Linux builds in the ["latest"](https://github.com/cvc5/cvc5/releases/tag/latest) release page. We would appreciate it if you could test the statically linked cvc5 binary (cvc5-Linux-arm64-static-*). If the binary works as expected, we will provide it in upcoming releases. Thank you.

**Reply by robby-phd** [reply]
Works for me. Thanks!

**Reply by daniel-larraz** [reply]
Following up on this: the first stable release of cvc5 that includes pre-compiled Linux arm64 binaries has been published as version [1.2.0](https://github.com/cvc5/cvc5/releases/tag/cvc5-1.2.0).
